Documentation/guides/building-apps/build-items.md

```diff
@@ -370,6 +370,11 @@ Starting in Xamarin.Android 11.3, the following MSBuild metadata is supported:
     online documentation.  Only one style is currently supported:
     `developer.android.com/reference@2020-Nov`.
 
+Starting in Xamarin.Android 12.1, the following MSBuild metadata is supported:
```

Comment:
Visual Studio 17.1, as found in the d17-1 branch, is Xamarin.Android 12.2, and this feature won't be available until the following release, Xamarin.Android *12.3*. :-)

Reply:
Oops I was thinking about the target framework version rather than product version 🤦